File: app/src/main/java/org/mifos/selfserviceapp/ui/activities/LoanApplicationActivity.java
package org.mifos.selfserviceapp.ui.activities;

import android.os.Bundle;

import org.mifos.selfserviceapp.R;
import org.mifos.selfserviceapp.ui.activities.base.BaseActivity;
import org.mifos.selfserviceapp.ui.enums.LoanState;
import org.mifos.selfserviceapp.ui.fragments.LoanApplicationFragment;

public class LoanApplicationActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_loan_application);

        replaceFragment(LoanApplicationFragment.newInstance(LoanState.CREATE), false,
                R.id.container);
        showBackButton();
    }
}

File: app/src/main/java/org/mifos/mobile/ui/activities/LoanApplicationActivity.kt
package org.mifos.mobile.ui.activities

import android.os.Bundle
import org.mifos.mobile.R
import org.mifos.mobile.databinding.ActivityLoanApplicationBinding
import org.mifos.mobile.ui.activities.base.BaseActivity
import org.mifos.mobile.ui.enums.LoanState
import org.mifos.mobile.ui.loan_account_application.LoanApplicationFragment

class LoanApplicationActivity : BaseActivity() {

    private lateinit var binding: ActivityLoanApplicationBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityLoanApplicationBinding.inflate(layoutInflater)
        setContentView(binding.root)
        if (savedInstanceState == null) {
            replaceFragment(
                LoanApplicationFragment.newInstance(LoanState.CREATE),
                false,
                R.id.container,
            )
        }
        showBackButton()
    }
}
